libs/src/services/creatorNode/index.js

```diff
@@ -45,15 +45,20 @@ class CreatorNode {
    * Pulls off the user's clock value from a creator node endpoint and the user's wallet address.
    * @param {string} endpoint content node endpoint
    * @param {string} wallet user wallet address
+   * @param {number} timeout max time alloted for clock request
+   * @param {Object?} [params={}] optional query string params
    */
-  static async getClockValue (endpoint, wallet, timeout) {
+  static async getClockValue (endpoint, wallet, timeout, params = {}) {
+    let baseReq = {
+      url: `/users/clock_status/${wallet}`,
+      method: 'get',
+      baseURL: endpoint,
+      timeout
+    }
+    baseReq = params && Object.keys(params).length > 0 ? { baseReq, params } : baseReq
```

Comment:
don't think this `{ baseReq, params }` assignment works as intended, you need to destructure ie `{ ...baseReq, ...params }` i think

did you test this flow?

Reply:
oh good catch -- I had updated this in my test script. updating